src/utils/utils.js

```diff
@@ -66,4 +66,20 @@ const releaseDates = [{ value: 'presence', label: 'Presence Of' }, { value: 'abs
 
 const curateFilters = [{ value: 'licensed', label: 'Licensed' }, { value: 'described', label: 'Described' }]
 
-export { setIfValue, difference, customLicenseIds, asObject, sorts, licenses, sources, releaseDates, curateFilters }
+const isMobile = window.innerWidth < 991
```

Comment:
That's true. This is was not listening for screen size changes. I assumed that was not necessary since, for example, the iPhone X height is 812px.

But I've made the change now